packages/terra-embedded-content-consumer/src/terra-dev-site/doc/embedded-content-consumer/EmbeddedContentConsumer.1.doc.jsx

```diff
@@ -32,35 +28,25 @@ const DocPage = () => (
     srcPath={`https://github.com/cerner/terra-framework/tree/master/packages/${name}`}
     examples={[
       {
-        title: 'Basic Embedded Content',
+        title: 'Basic Embedded Content Consumer',
         example: <BasicConsumer />,
         source: BasicConsumerSrc,
       },
       {
-        title: 'Filled Embedded Content',
```

Comment:
Removed the Fill example. This example was dead code left over from the initial PR to introduce the embedded-content-consumer. Here is the PR commit that removed the fill prop from the initial review: https://github.com/cerner/terra-core/pull/737/commits/65573eb98e0ca97b2d10448a7a3dc0ac08ec3f7a